README.zh-CN.md

```diff
@@ -101,30 +96,32 @@ vim conf/eventmesh.properties
 eventMesh.storage.plugin.type=rocketmq
 ```
 
-执行`start.sh`脚本启动EventMesh Runtime服务器。
+执行 `start.sh` 脚本启动 EventMesh Runtime 服务器。
 ```
 bash bin/start.sh
 ```
-如果看到`EventMeshTCPServer[port=10000] started....`, 则说明设置成功。
 
 查看输出日志:
 ```
-cd /root/apache-eventmesh-1.10.0/logs
-tail -f eventmesh.out
+tail -n 50 -f logs/eventmesh.out
 ```
 
+当日志输出 `server state:RUNNING`，则代表 EventMesh Runtime 启动成功了。
+
 停止:
 ```
 bash bin/stop.sh
 ```
 
-### 在 Docker 中运行 EventMesh Runtime  
+脚本打印 `shutdown server ok!` 时，代表 EventMesh Runtime 已停止。
 
-#### 1.获取EventMesh镜像
+### 在 Docker 中运行 EventMesh Runtime
 
-首先，你可以打开一个命令行，并且使用下面的`pull`命令从[Docker Hub](https://hub.docker.com)中下载最新发布的[EventMesh](https://hub.docker.com/r/apache/eventmesh)。
+#### 1.获取 EventMesh 镜像
+
+首先，你可以打开一个命令行，并且使用下面的 `pull` 命令从 [Docker Hub](https://hub.docker.com) 中下载最新发布的 [EventMesh](https://hub.docker.com/r/apache/eventmesh)。
 ```
```

Comment:
This narrative could be deleted.

Please add a new line between paragraphs and code blocks when writting markdown.

Reply:
You this description I did not understand, is to put this sentence ' First of all, you can open a command line, and use the following ' pull ' command to download the latest release from the Docker Hub ' delete ? No need to remind ? I rewrote it in a recent update.

Reply:
I apologize for any confusion caused. What I mean is that this paragraph is unnecessary and can be deleted. Additionally, I have pointed out a common formatting issue in this markdown document.

Reply:
I still don 't understand, you said the format problem I have solved, you want to pull the mirror this paragraph description delete is it ? 

<img width="1062" alt="image" src="https://github.com/apache/eventmesh/assets/91315508/0af24d47-383e-40e2-a7d5-d7508e595936">

Reply:
resolved